app/controllers/students_controller.rb

```diff
@@ -34,6 +34,7 @@ def show
       event_note_types_index: EventNoteSerializer.event_note_types_index,
       educators_index: Educator.to_index,
       access: student.latest_access_results,
+      transition_notes: student.transition_notes,
```

Comment:
Will this send down restricted notes, regardless of whether the user has access to view restricted notes or to this feature?  Maybe add an authorization check function for this in `authorizer`.

Reply:
👍